### DISCUSSION TITLE
Install build software and tool on own VMs

### DISCUSSION THREAD

**Discussion by WAG10** [actions/virtual-environments#9717] [Installed software]
Hi team,
We have our own VM which is security hardened as per company policies. We would like to know if we can install all the build tools and software(which gets installed when we create VM using this repo) in it using this repo. If yes, how can we do that?(like steps or scripts to be run for the same)

**Comment by Alexey-Ayupov** [comment]
Hello @WAG10 . Yes you can use any scripts from this repository to install dedicated software. However if you would like to do it on top of existed VM, probably you have to write some wrapper script to handle sequence of installation scripts. In our case the Packer handle this.
Another way is to start scripts one by one manually.

**Reply by WAG10** [reply]
Hi @Alexey-Ayupov , Thanks a lot, Can you please guide me to script locations. I am actually naïve into this :)
